src/init.cpp

```diff
@@ -1211,114 +1211,14 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
     assert(!node.connman);
     node.connman = std::make_unique<CConnman>(GetRand(std::numeric_limits<uint64_t>::max()), GetRand(std::numeric_limits<uint64_t>::max()), *node.addrman, args.GetBoolArg("-networkactive", true));
 
+    // ********************************************************* Step 7: load block chain
```

Comment:
https://github.com/bitcoin/bitcoin/pull/22564/commits/3de7842f50c44e6bed675048737e19a1d8ca5fa4

This is most easily reviewed with `--color-move=dimmed_zebra`.

Given that init reordering is notoriously tricky, we should probably be sure to test invocation of `bitcoind` with each flag that was moved here.

Reply:
Yes, I think I'm going to work a bit more on this commit to see if I can make it more easily reviewable.